README.md

```diff
@@ -84,6 +84,26 @@ The current paragraph — what would be selected if you typed `vip` — is autom
 
 To control exactly what is sent, you can manually select text before calling `vim-slime`.
 
+## Vim-like mappings
+
+To use vim-style mappings:
+
+```vim
+"disables mappings from previous usage section
+let g:slime_no_mappings = 1
+
+"send visual selection
+xmap <leader>s <Plug>SlimeRegionSend
+
```

Comment:
This README used to be pretty long, so I had to make an editorial choice as to what I kept.

In the end, I tried to push config and tweaks to [Advanced Configuration](https://github.com/jpalardy/vim-slime/blob/main/assets/doc/advanced.md)

If you want to add a pointer in the [Usage](https://github.com/jpalardy/vim-slime?tab=readme-ov-file#usage) section, pointing the Advanced Configuration — I think that's the way to go.

(and this text can be integrated there)